### PR TITLE
Add virtual storage pool edit action

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

This adds actions to the virtual storage management page.

## GUI diff

Added buttons for the storage pool actions

system_details_salt_virtual_pools

- [X] **DONE**

## Documentation
- [uyuni-docs](uyuni-project/uyuni-docs#52) PR or issue was created

- [X] **DONE**

## Test coverage
- Unit tests were added
- Cucumber tests were added

- [X] **DONE**

## Links

SUSE/spacewalk#7037

- [X] **DONE**

## Changelogs

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [x] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_lint_checkstyle"		 
- [ ] Re-run test "java_pgsql_tests"		 
- [ ] Re-run test "ruby_rubocop"
- [ ] Re-run test "schema_migration_test_oracle"
- [ ] Re-run test "schema_migration_test_pgsql"		 
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"		 
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Comment by cbosdo** [comment]
Addressing the actions part of issue SUSE/spacewalk#7037

File: testsuite/features/secondary/minkvm_guests.feature [hunk 1]
```diff
@@ -194,6 +194,66 @@ Feature: Be able to manage KVM virtual machines via the GUI
...
+  Scenario: Create a virtual storage pool for KVM
+    Given I am on the "Virtualization" page of this "kvm_server"
+    When I follow "Storage"
+    And I follow "Create Pool"
+    And I wait until I see "General" text
+    And I select "dir" from "type"
+    And I enter "test-pool1" as "name"
+    And I uncheck "autostart"
+    And I enter "/var/lib/libvirt/images/test-pool1" as "target_path"
+    And I enter "0755" as "target_mode"
```
Comment (Bischoff) [inline testsuite/features/secondary/minkvm_guests.feature]: what about using a notation a bit more modern ? rwxr-xr-x

:trollface:

Comment (cbosdo) [reply testsuite/features/secondary/minkvm_guests.feature]: so far it's passed as-is to libvirt :)

**Comment by cbosdo** [comment]
> LGTM (sorry for the small troll, could not resist)

I see :)

**Comment by cbosdo** [comment]
This pull request requires saltstack/salt#55814

**Comment by cbosdo** [comment]
This PR has been split into a number of smaller ones:

* #1979 - JS refactoring
* ~~#1980 - Java refactoring~~
* #1982 - virtual pool capabilities REST API
* #1983 - virtual pool definition REST API
* #1984 - refresh action
* #1985 - start action
* #1986 - stop action
* #1987 - delete action
* #1988 - create action
* This one - edit action

**Pull request merged**